### DISCUSSION TITLE
How did you get so many stars on your repo?

### DISCUSSION THREAD

**Discussion by kun-codes** [rohankishore/Aura-Text#96] [General]
I too want some github stars on my repo over here https://github.com/kun-codes/pomodoro-task-app.py

I want to learn your way of getting your project noticed

**Comment by rohankishore** [comment]
I just shared my project in r/python in reddit. And also, most of the stars are from github explore page

**Comment by kun-codes** [comment]
how do you know that most stars are from the github explore page

**Comment by rohankishore** [comment]
Because my project was featured in the explore. And also, I haven't shared my project to reddit in a while

**Comment by kun-codes** [comment]
Thanks

**Comment by rohankishore** [comment]
You're always welcome:)
